types/artifacts.d.ts

```diff
@@ -182,6 +184,28 @@ declare global {
         params: {name: string; value: string}[];
       }
 
+      export interface IFrameElement {
+        /** The `id` attribute of the iframe. */
+        id: string,
+        /** The `src` attribute of the iframe. */
+        src: string,
+        /** The iframe's ClientRect. @see https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect */
+        clientRect: {
+          top: number;
+          bottom: number;
+          left: number;
+          right: number;
+          width: number;
+          height: number;
+        },
+        /** The pixel area iframe. */
+        pixelArea: number,
+        /** If the iframe or an ancestor of the iframe is fixed in position. */
+        isPositionFixed: boolean,
+        /** The Frame of the iframe. @see https://chromedevtools.github.io/devtools-protocol/tot/Page#type-Frame */
+        frame?: Crdp.Page.Frame | null,
```

Comment:
could this be just `Crdp.Page.Frame | null` or `Crdp.Page.Frame | undefined`? current is `Crdp.Page.Frame | null | undefined`

Reply:
SGTM, went with `frame?: Crdp.Page.Frame` as that seems to follow the precedent on that file.